Platform/de.lproj/Localizable.strings

```diff
@@ -829,13 +829,13 @@
 "The backend for this configuration is not supported." = "Diese Konfiguration wird auf dieser Plattform nicht unterstützt.";
 
 /* UTMConfiguration */
-"The drive '%@' already exists and cannot be created." = "Laufwerk konnte nicht erstellt werden: ein Laufwerk %@ existiert bereits.";
+"The drive '%@' already exists and cannot be created." = "Laufwerk konnte nicht erstellt werden: das Laufwerk %@ existiert bereits.";
 
 /* UTMDownloadSupportToolsTaskError */
 "The guest support tools have already been mounted." = "Die Gast-Unterstützung ist bereits eingelegt.";
 
 /* UTMAppleConfiguration */
-"The host operating system needs to be updated to support one or more features requested by the guest." = "Ihr Host-Betriebssystem muss aktualisiert werden, um eines oder mehrere von der VM erwartete Features freizuschalten.";
+"The host operating system needs to be updated to support one or more features requested by the guest." = "Ihr Host-Betriebssystem muss aktualisiert werden, um eines oder mehrere von der VM erwarteten Features freizuschalten.";
```

Comment:
⚠️ To me the grammar of the last part of the sentence sounds odd, the grammar seems off. Additionally "erwartet" is a good translation for "expected", but less so for "requested", hence I suggest use of "angeforderte".

What do you think?

```suggestion
"The host operating system needs to be updated to support one or more features requested by the guest." = "Ihr Host-Betriebssystem muss aktualisiert werden, um ein oder mehrere von der VM angeforderte Features freizuschalten.";
```